src/backend/utils/misc/guc.c

```diff
@@ -7137,10 +7137,20 @@ parse_and_validate_value(struct config_generic *record,
 
 				if (!parse_bool(value, &newval->boolval))
 				{
-					ereport(elevel,
+					if (strstr(name, "babelfishpg_tsql") != NULL) {
```

Comment:
Let's not make any engine change. Our strategy is to minimize engine changes and I don't think it's worth to make a change here just to produce error messages.